junifer/datagrabber/hcp.py

```diff
@@ -199,3 +199,8 @@ def __init__(
             uri=uri,
             rootdir=rootdir,
         )
+
+    @property
```

Comment:
Do we need to have this here? I thought the pattern one would have the default (`False`)

Reply:
Ah, sorry, this is the Datalad one, this should return `True` then.

Reply:
I believe this is ok then.